hubblestack/extmods/modules/audit.py

```diff
@@ -109,342 +103,285 @@
 
 log = logging.getLogger(__name__)
 
-hubble_status = HubbleStatus(__name__, 'top', 'audit')
-
-__audit__ = None
+hubble_status = HubbleStatus(__name__, 'top', 'run')
+BASE_DIR_NOVA_PROFILES = 'hubblestack_nova_profiles_v2'
 
+CHECK_STATUS = {
+    'Success': 'Success',
+    'Failure': 'Failure',
+    'Skipped': 'Skipped',
+    'Error': 'Error'
+}
 
 @hubble_status.watch
-def audit(audit_files=None,
-          tags='*',
-          labels=None,
-          verbose=True,
-          show_success=True):
+def run(audit_files=None,
+        tags='*',
+        labels=None,
+        verbose=None,
+        show_compliance=None):
     """
-    Execute one or more audit files, and return the cumulative results.
-
     :param audit_files:
-        Which audit files to execute. Can contain multiple files (list or
-        comma-separated).
+        Profile to execute. Can have one or more files 
+        (python list or comma separated) (default: {None})
     :param tags:
         Can be used to target a subset of tags via glob targeting.
     :param labels:
-        Only run the checks with the given label(s). Can contain multiple
-        labels (comma-separated). If multiple labels are provided, a check
-        with any label in the list will be run.
-    :param verbose:
+        Tests with matching labels are executed. If multiple labels are passed, 
+        then tests which have all those labels are executed.
+    :param verbose: 
         True by default. If set to False, results will be trimmed to just tags
         and descriptions.
-    :param show_success:
-        Whether to show successes/skipped or just failures. Defaults to True
+    :param show_compliance:
+        Whether to show compliance with results or not
     :return:
         Returns dictionary with Success, Skipped, and Failure keys and the
         results of the checks
     """
-    ret = {'Success': [],
-           'Failure': [],
-           'Skipped': []}
-
+    # if audit_files is None:
+    #     return top(verbose=verbose,
+    #                show_compliance=show_compliance,
+    #                labels=labels)
+
+    audit_runner = runner_factory.get_audit_runner(__salt__, __grains__, __opts__)
+
+    # categories of results
+    result_dict = {
+        'Success': [],
+        'Failure': [],
+        'Error': [],
+        'Skipped': [],
+    }
+    combined_dict = {}
+
+    if verbose is None:
+        verbose = __salt__['config.get']('hubblestack:nova:verbose', False)
+    if show_compliance is None:
+        show_compliance = __salt__['config.get']('hubblestack:nova:show_compliance', True)
```

Comment:
Same as above

Reply:
We wil do it later, as this change is in config file too.